device/mellanox/x86_64-mlnx_msn2700-r0/ACS-MSN2700/buffers_defaults_t0.j2

```diff
@@ -81,10 +81,10 @@
         "{{ port_names }}|3-4": {
             "profile" : "[BUFFER_PROFILE|egress_lossless_profile]"
         },
-        "{{ port_names }}|0-1": {
+        "{{ port_names }}|0-2": {
```

Comment:
Do you still need additional PGs configured here
https://github.com/Azure/sonic-buildimage/blob/master/files/build_templates/buffers_config.j2#L134?
It is global template and will affect all devices and topologies.
And this config will fail on t1/t1-lag (which use 300m cables) on split ports

Reply:
ok, found in [another PR](https://github.com/Azure/sonic-buildimage/pull/2650)
maybe it makes sense to do all changes in one PR?

Reply:
Ingress change won't affected the egress change that is in the other PR. The two PRs are independent.

https://github.com/Azure/sonic-buildimage/pull/2651/files#r263354375